test/unit/range/views/single_pass_input_test.cpp

```diff
@@ -180,12 +180,15 @@ TYPED_TEST(single_pass_input, view_iterate)
     }
     else
     {
+        // Single pass input is only movable.
         seqan3::detail::single_pass_input_view view{p};
+
         TypeParam tmp{this->cmp_data};
-        auto zipper = seqan3::views::zip(tmp, std::move(view));
-        for (auto it = zipper.begin(); it != zipper.end(); ++it)
+        auto tmp_it = tmp.begin();
+        for (auto && elem : view)
         {
-            EXPECT_EQ(std::get<0>(*it), std::get<1>(*it));
+            EXPECT_EQ(elem, *tmp_it);
+            ++tmp_it;
         }
```

Comment:
Can you check if `EXPECT_RANGE_EQ` works?
something like `EXPECT_RANGE_EQ(tmp, std::move(view))`

Reply:
Awesome! Works perfectly. I used it without calling `std::move`. Is there any pros/cons to this?

Reply:
No, just less clutter and it covers some different cases.
You can check the implementation if you want to: test/include/seqan3/test/expect_range_eq.hpp